app/routes/crate/index.js

```diff
@@ -3,4 +3,12 @@ import VersionRoute from './version';
 export default class CrateIndexRoute extends VersionRoute {
   controllerName = 'crate.version';
   templateName = 'crate/version';
+
+  setupController(controller) {
+    super.setupController(...arguments);
+
+    controller.loadDownloadsTask.perform().catch(() => {
+      // ignored
+    });
+  }
```

Comment:
why here and not in the parent class?

Reply:
Good catch! You're right, `crate.version` route needs this too. I'll make the change.

Reply:
Updated!